src/tutorial/source.rst

```diff
@@ -0,0 +1,83 @@
+.. py:currentmodule:: palace
+
+Source manipulation
+===================
+
+We have created a source in the last section.
+As said previously, its properties can be manipulated
+to create wanted effects.
+
+Moving the source
+-----------------
+
+Changing :py:meth:`Source.position` is one of the most noticeable,
+but first, we have to enable spatialization via :py:attr:`Source.spatialize`.
+
+.. code-block:: python
+
+   from time import sleep
+   from palace import Device, Context, Source, decode
+
+   filename = 'some_audio.ogg'
+   with Device() as dev, Context(dev) as ctx:
+       with Source() as src:
+           src.spatialize = True
+
+           dec = decode(filename)
```

Comment:
Redundant whitespace and redundant `filename` declaration.  Also I believe 3 context manages fit in one line.  Unless we're running out of horizontal space (79-col limit), please use full words like `device`, `context` and `source`.